PSReadLine/PSReadLine.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Maybe restore the original encoding too?

Reply:
Done.